cmd/sandbox-ce/cmd/config.go

```diff
@@ -0,0 +1,216 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+	"gopkg.in/src-d/go-cli.v0"
+)
+
+const orgName = "smacker"
+const repoName = "superset-compose"
+const defaultConfigName = "__default__"
+const composerConfigFileURL = "https://raw.githubusercontent.com/%s/%s/%s/docker-compose.yml"
+
+type configCmd struct {
+	cli.PlainCommand `name:"config" short-description:"Config"`
+}
+
+type configDownloadCmd struct {
+	Command `name:"download" short-description:"Download"`
+
+	Revision string `short:"r" long:"revision" description:"revision to download" default:"master"`
+	URL      string `short:"u" long:"url" description:"url of the config to download"`
+}
+
+func (c *configDownloadCmd) Execute(args []string) error {
+	var err error
+	var confURL, outPath string
+
+	if c.URL != "" {
+		confURL = c.URL
+		outPath, err = getConfigPath(strconv.Itoa(int(time.Now().Unix())))
+	} else {
+		confURL = getComposerConfFileURL(c.Revision)
+		outPath, err = getConfigPath(c.Revision)
+	}
+
+	if err != nil {
+		return err
+	}
+
+	err = c.downloadConfig(confURL, outPath)
+	if err != nil {
+		return err
+	}
+
+	return setDefaultConfig(outPath)
+}
+
+func (c *configDownloadCmd) downloadConfig(url, path string) error {
```

Comment:
Maybe we should reuse the same code for both initial download (that is already in master) and for download command.

Reply:
refactored, see https://github.com/src-d/superset-compose/pull/29/files#diff-480c3c3933838ac5fb6eebb456127ab1R65